doc/development/plugins.rst

```diff
@@ -97,169 +92,165 @@ as well as potential further capabilities, by providing the following class attr
 
     operations = {"CNOT", "PauliX"}
 
-  This is used to decide whether an operation is supported by your device in the
-  default implementation of the public method :meth:`.Device.supports_operation`.
-
-* :attr:`.Device.observables`: set of the supported PennyLane observables as strings, e.g.,
-
-  .. code-block:: python
-
-    observables = {"PauliX", "PauliZ", "Hadamard", "Hermitian"}
-
-  This is used to decide whether an observable is supported by your device in the
-  default implementation of the public method :meth:`.Device.supports_observable`.
+  See :doc:`/introduction/operations` for a full list of operations
+  supported by PennyLane.
 
 * :attr:`.Device._capabilities`: a dictionary containing information about the capabilities of
   the device. Keys currently supported include:
 
   * ``'model'`` (*str*): either ``'qubit'`` or ``'CV'``.
 
-  * ``'tensor_observables'`` (*bool*): ``True`` if the device supports measuring tensor products
-    of observables, ``False`` otherwise.
-
-For a better idea of how to best implement :attr:`.Device.operations` and
-:attr:`.Device.observables`, refer to the two reference plugins.
+  * ``"inverse_operations"`` (*bool*): ``True`` if the device supports
+    applying the inverse of operations. Operations which should be inverted
+    have ``operation.inverse == True``.
```

Comment:
Maybe more specific than "have"? ("have the attribute"?)

Reply:
```suggestion
    have the property ``operation.inverse == True``.
```